src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/Terminal.tsx

```diff
@@ -109,8 +109,7 @@ function TerminalView(terminalViewProps: TerminalViewProps) {
             _socket.send(JSON.stringify(startData));
 
             let dim = _fitAddon.proposeDimensions();
-            const resize = { Op: 'resize', Cols: dim.cols, Rows: dim.rows };
-
+            const resize = { Op: 'resize', Cols: dim?.cols, Rows: dim?.rows };
```

Comment:
Cols and Rows will be undefined in case of no dim. Is it expected?

Reply:
As per discussion, we are reverting this changes & will be covered if encountered in future